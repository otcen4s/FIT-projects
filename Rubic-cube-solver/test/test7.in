116
116
166
322 533 233 442
422 533 244 553
544 553 455 422
166
166
116