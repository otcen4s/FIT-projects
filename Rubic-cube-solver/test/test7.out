116
116
166
322 533 233 442
422 533 244 553
544 553 455 422
166
166
116

116
116
166
322 533 233 442
422 533 244 553
553 455 422 544
666
661
111

116
116
166
322 533 233 442
422 533 244 553
455 422 544 553
611
661
661

116
116
554
225 133 233 446
225 133 244 556
344 622 544 551
233
661
661

116
116
116
554 333 233 444
224 333 244 555
223 222 544 555
661
661
661

114
114
113
551 233 633 444
221 233 644 555
221 233 644 555
665
662
662

111
111
111
555 222 333 444
222 333 444 555
222 333 444 555
666
666
666

111
111
111
222 333 444 555
222 333 444 555
222 333 444 555
666
666
666

